wls-gui-wahllokalsystem/tests/stores/wahlenStore.spec.ts

```diff
@@ -375,4 +375,71 @@ describe("wahlenStore.ts", () => {
       expect(unitUnderTest.isBeanstandeteWahlbriefeSaving).toBe(false);
     });
   });
+
+  describe("summeGueltigerWahlbriefe", () => {
+    it("should_calculateSummeGueltigerWahlbriefe_whenBeanstandeteWahlbriefeIsEmpty", async () => {
```

Comment:
bei allen anderen wahlenstore tests auch noch bitte anpassen
```suggestion
    it("should_calculateSummeGueltigerWahlbriefe_when_BeanstandeteWahlbriefeIsEmpty", async () => {
```

Reply:
[78f7990](https://github.com/it-at-m/Wahllokalsystem/pull/1651/commits/78f7990d1de8e25b4fcb2385b7e48cd9defb6272)